### PR TITLE
chore: Add report for error case (WEBAPP-4817)

### PR DESCRIPTION
**Pull request by gregor**

### CODE REVIEW

File: app/script/conversation/ConversationRepository.js [hunk 1]
```diff
@@ -2291,9 +2294,20 @@ z.conversation.ConversationRepository = class ConversationRepository {
                   ? z.string.modalConversationNewDeviceHeadlineYou
                   : z.string.modalConversationNewDeviceHeadlineOne;
               } else {
-                const log = `Granting '${consentType}' message '${messageType}' in '${conversationId}' needs user ids`;
+                const log = `Granting message '${messageType}' in '${conversationId}' for '${consentType}'  needs ids`;
```
Comment (bennycode) [inline app/script/conversation/ConversationRepository.js]: `needs ids` -> `needs user ids`?

Comment (gregor) [reply app/script/conversation/ConversationRepository.js]: Need to reword this. It no longer fits into a single line if I add the word user with prettier. The message is obviously not concise enough in that case.

Comment (bennycode) [reply app/script/conversation/ConversationRepository.js]: There are also 2 spaces before `needs`.

File: app/script/conversation/ClientMismatchHandler.js [hunk 2]
```diff
@@ -157,7 +157,10 @@ z.conversation.ClientMismatchHandler = class ClientMismatchHandler {
 
         const _removeRedundantUser = userId => {
           if (conversationEntity && conversationEntity.is_group()) {
-            conversationEntity.participating_user_ids.remove(userId);
+            const timeOffset = this.conversationRepository.timeOffset;
+            const event = z.conversation.EventBuilder.buildMemberLeave(conversationEntity, userId, timeOffset);
+
+            amplify.publish(z.event.WebApp.EVENT.INJECT, event);
```
Comment (gregor) [inline app/script/conversation/ClientMismatchHandler.js]: Potential fix for the underlying issue

**Pull request merged**